src/components/TaskTracker/Settings.tsx

```diff
@@ -36,10 +36,13 @@ export const Settings = ({ setOpenSettings, Task }) => {
   }
 
   return (
-    <div className="mb-2 w-full space-y-2 rounded-lg border border-gray-200 bg-white py-2 px-1 text-gray-800 shadow-md dark:border-gray-700 dark:bg-gray-800 dark:text-gray-300 ">
+    <div className="mb-6 mt-2 space-y-2 w-full rounded-lg border border-gray-200 bg-white py-2 px-1 text-gray-800 shadow-md dark:border-gray-700 dark:bg-gray-800 dark:text-gray-300 ">
       <div className="flex">
+
+        {/* We need to cancel drag here, if the user is selecting some text
```

Comment:
IDK if this comment is needed

Reply:
Indeed. Seems less important now. I've removed it. 

Resolved in c9b6320109809b08f43a9f577ed4c102ddc1146d